### PR TITLE
WIP: Introduce changelog with basic information

### PR DESCRIPTION
**Pull request by ingvagabund**
Worth introducing changelog that describes important a high-level changes.

@enxebre @frobware @paulfantom @bison @spangenberg PTAL

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+### Added
+- Allow to specify availability zone in placement
+- Support for machine classes
+- Record machine events
+
+## [0.1.0] - 2018-10-17
+### Added
+
+- Self-standing binary allows tp bootstrap Kubernetes cluster with cluster API stack deployed
```
Comment (bison) [inline CHANGELOG]: Typo? "to bootstrap"?

**Comment by paulfantom** [comment]
I could autogenerate one based on past PR topics if there is a need :smile:

**Comment by ingvagabund** [comment]
> I could autogenerate one based on past PR topics if there is a need. Later we could shape it smile

no

**Pull request closed without merging**